end_effectors/ft_sensors/ati_mini45.urdf.xacro

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>
+<robot xmlns:xacro="http://ros.org/wiki/xacro" name="ati_mini45">
+  <xacro:include filename="$(find franka_description)/end_effectors/common/utils.xacro" />
+  <xacro:include filename="$(find franka_description)/end_effectors/common/ee_with_one_link.xacro" />
+  <xacro:macro name="ati_mini45"
+    params="prefix parent_link rpy_sensor:='0 0 0' xyz_sensor:='0 0 0' ip_address rdt_sampling_rate internal_filter_rate use_hardware_biasing sensor_inertials safety_distance:=0 gazebo:=false description_pkg:=franka_description with_sc:=false">
+
+    <!-- length and radius of the sensor with mounting plates -->
+    <xacro:property name="radius" value="0.0315" />
+    <xacro:property name="length" value="0.0327" />
+    <xacro:property name="ee_inertials" value="${sensor_inertials}" />
+
+    <joint name="${prefix}hand_joint" type="fixed">
```

Comment:
I would prefer a name that is specific to the sensor itself.
```suggestion
    <joint name="${prefix}ati_mini45_base" type="fixed">
```
or `ati_mini45_base_joint` or just `ati_mini45_joint`.

Reply:
Fixed